### PR TITLE
Revert relayed v3

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: data/transaction.go [hunk 1]
```diff
@@ -11,24 +11,23 @@ import (
...
+	GasPrice          uint64 `json:"gasPrice"`
+	GasLimit          uint64 `json:"gasLimit"`
+	Data              []byte `json:"data,omitempty"`
+	Signature         string `json:"signature,omitempty"`
+	ChainID           string `json:"chainID"`
+	Version           uint32 `json:"version"`
+	Options           uint32 `json:"options,omitempty"`
+	GuardianAddr      string `json:"guardian,omitempty"`
+	GuardianSignature string `json:"guardianSignature,omitempty"`
+	Relayer           string `json:"relayer,omitempty"`
```
Comment (sasurobert) [inline data/transaction.go:30]: why do you keep the relayer here ?

Comment (AdoAdoAdo) [inline data/transaction.go:30]: relayer can be removed as well

Comment (sstanculeanu) [reply data/transaction.go]: pushed

Comment (sstanculeanu) [reply data/transaction.go]: pushed

**Review by AdoAdoAdo — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**